tests/klay_blockchain_test.go

```diff
@@ -89,7 +89,7 @@ func TestSimpleBlockchain(t *testing.T) {
 	}
 }
 
-func newBlockchain(t *testing.T) (*node.Node, *cn.CN, *TestAccountType, *big.Int, string) {
+func newBlockchain(t *testing.T, config *params.ChainConfig) (*node.Node, *cn.CN, *TestAccountType, *big.Int, string) {
```

Comment:
This is exactly the same as `newBlockchainWithConfig()`. Are you going to remove `newBlockchainWithConfig()` later?

Reply:
Yes. I will